Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
+        private int id = 10;
```

Comment:
The `branchName` variable should have an explicit type. Please define it as `private string branchName = "Dev";`. This will improve readability and maintainability.